src/evict/evict_page.c

```diff
@@ -134,6 +134,9 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
      */
     time_start = 0;
     if (LF_ISSET(WT_EVICT_CALL_URGENT)) {
+        /* Set this page to be evicted as soon as possible. */
+        __wt_page_evict_soon(session, ref);
```

Comment:
This change looks innocuous, but I suspect it's the most significant here.
It seems right to me, as in I don't know why we'd urgently evict something and not mark it for immediate eviction as well, but I believe this code path is common, so it's a place to watch for fallout.